AceGUI-3.0/AceGUI-3.0.lua

```diff
@@ -772,7 +772,7 @@ AceGUI:RegisterLayout("Flow",
 				rowoffset = child.alignoffset or (rowheight / 2)
 				rowstartoffset = rowoffset
 			elseif child.width == "relative" then
-				safelayoutcall(child, "SetWidth", width * child.relWidth)
+				safelayoutcall(child, "SetWidth", framewidth)
```

Comment:
There's already
```lua
if child.width == "relative" then
    framewidth = width * child.relWidth
end
```
at the top of the function, felt like a waste not to actually use it 😅